examples/ui.apps/src/main/content/jcr_root/apps/cif-components-examples/config/com.adobe.cq.commerce.graphql.client.impl.GraphqlClientImpl-examples.config

```diff
@@ -1,5 +1,5 @@
 identifier="examples"
-url="https://localhost:8443/apps/cif-components-examples/graphql"
+url="$[env:CIF_MOCK_GRAPHQL_ENDPOINT]"
```

Comment:
Why not using `COMMERCE_ENDPOINT` here? Yes it is a mock GraphQl server not a real on, but with using COMMERCE_ENDPOINT we can also deploy this together with the CIF Add-On.

Reply:
I'd really keep this separate, mainly because the mock server currently only supports GraphQL requests for the components (I have no idea what responses you'd get for example for the Omnisearch), and also to really keep that separate from the production configuration. Once can easily set the two variables, but I'd not mix a mock config with a production config.